motr/io_nw_xfer.c

```diff
@@ -1208,6 +1209,8 @@ static int target_ioreq_init(struct target_ioreq    *ti,
 		ti->ti_goff_ivec.iv_vec.v_nr = 0;
 		if (rc != 0)
 			goto fail;
+	} else {
+		ti->ti_goff_ivec.iv_index = NULL;
 	}
```

Comment:
Coding standard, bracket may not be needed

Reply:
done